assets/javascripts/discourse/widgets/discourse-reactions-actions.js.es6

```diff
@@ -359,11 +363,27 @@ export default createWidget("discourse-reactions-actions", {
       });
 
       if (!isAvailable) {
-        post.reactions.push({
+        const newReaction = {
           id: attrs.reaction,
           type: "emoji",
           count: 1
-        });
+        };
+
+        const tempReactions = Object.assign([], post.reactions);
+
+        tempReactions.push(newReaction);
+
+        const newReactionIndex = tempReactions
+          .sort((reaction1, reaction2) =>
```

Comment:
all this code is not readable at all

Reply:
I know but we want to sort array by 2 variables. One is count and another is string. In order to do that this was the best solution i found😅 can you suggest any improvements?